WordPress/gutenberg
# [RNMobile] Use original repository for fetching `react-native-get-random-values`

## What?

Use the original repository for fetching the package `react-native-get-random-values` within `react-native-editor` instead of the forked repository.

## Why?

This is part of an effort to stop using forked repositories for some of the `react-native-editor` dependencies. The idea is to remove the ones that only introduce logic for publishing the Android library in their forked repositories.

## How?

The `react-native-get-random-values` package now points to the `1.4.0` version published in NPM. Additionally, some modifications to the Android build configuration have been applied to use the dependency artifacts instead including the source code.

## Testing Instructions





**NOTE:** One of the uses of `react-native-get-random-values` package is related to obtaining a UUID for each block rendered, so it would be relatively easy to identify potential errors just by opening the editor and adding some blocks.

1. Open a post/editor in the app.
2. Observe that the editor is initialized properly without any errors.
3. Add some blocks and observe that work as expected.

## Screenshots or screencast 
N/A
